### DISCUSSION TITLE
Discussion section public on private repo

### DISCUSSION THREAD

**Discussion by 42atomys** [community/community#12169] [Discussions]
Hello to all,

today I was wondering if it was possible to make public the Discussions or some parts of the discussions on a private repository. 

The idea is to use the Discussions functionality on a code that has a private base (permanent or temporary, or even sponsorware) but allow to grow a community around the project. Example: Before its official launch for example, with the idea of making the project open-source in the coming time and let future developers share their ideas, or their feedback first 

A workaround exists: Create a public side repo without code to host only the discussion section, however it separates in my opinion. 

Maybe an idea to raise? 

**Make discussion categories public on private repo**

**Comment by jsoref** [comment]
GitHub does this. There are github private repositories in the `github` organization. What they've done is create a dummy repository `feedback` to use for community discussion.

This is clearly how they imagine others would do it as well. (And it's how I'd suggest doing it even if I didn't know they were doing this.)
